.github/workflows/build_test.yml

```diff
@@ -14,8 +15,9 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - {rosdistro: 'melodic', container: 'px4io/px4-dev-ros-melodic:2021-12-11'}
-          - {rosdistro: 'noetic', container: 'px4io/px4-dev-ros-noetic:2021-12-11'}
+          - {rosdistro: 'humble', container: 'osrf/ros:humble-desktop'}
```

Comment:
If you only have one set in the matrix, I think just hard code these. Matrix like this will only work if humble, iron, and rolling have no difference. The more common pattern is to create a unique branch to manage each ROS version and then do all dev on rolling.

Reply:
Right, my intention was have this in place for iron when we want to. Does it make sense to add iron now?

Reply:
Absolutely.